README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/red-hat-storage/cephci.svg?branch=master)](https://travis-ci.com/red-hat-storage/cephci)
+[![Build Status](https://github.com/red-hat-storage/cephci/workflows/tests/badge.svg)](https://github.com/red-hat-storage/cephci/actions)
```

Comment:
missing badge?

Reply:
I think we should remove this build badge, it actually does not imply that ceph build or unit tests for ceph-ci libs is passing. which i think it should. we just have some tox and pylint checks for now. It makes more sense when we actively maintain unit tests. This one (8ea01d04c45399c955a1a6517557cbcb0f0e8dbe ) was an enthusiastic one . :) 
What do you think @vasukulkarni and @ktdreyer

Reply:
i think we are okay to merge this.